lib/tasks/import/orgs_sites_hosts.rake

```diff
@@ -1,9 +1,14 @@
 require "transition/import/orgs_sites_hosts"
 
 namespace :import do
-  desc "Import all Organisations, and Sites and Hosts from the given filename or mask"
-  task :orgs_sites_hosts, [:filename_or_mask] => :environment do |_, args|
-    Transition::Import::OrgsSitesHosts.from_yaml!(args.filename_or_mask)
+  desc "Import all Organisations"
+  task organisations: :environment do
+    Transition::Import::Organisations.from_yaml!
+  end
+
+  desc "Import all Sites and Hosts from the given filename or mask"
```

Comment:
I _think_ we'll be able to remove this rake task (and the associated code) too, since we will no longer need to import anything from Transition Config.

But this looks totally fine if you're planning on the code removal being done in two stages.

Reply:
I'd rather do it in two stages, since it looks like we'll be able to get rid of `.notmodules` and related things as well 🤔

Reply:
Seems like a good approach 👍